dash-pipeline/SAI/templates/saifixedapis.cpp.j2

```diff
@@ -79,14 +79,14 @@ sai_status_t sai_get_switch_attribute(
             return SAI_STATUS_SUCCESS;
         
         case SAI_SWITCH_ATTR_PORT_LIST:
-            // make a tmp port list, saiserver will free the memory
-            objlist = (sai_object_id_t *)malloc(sizeof(port_list));
```

Comment:
this is memory leak, caller should provide memory for placing objects in the .list member

Reply:
Good catch!